uxarray/io/_mpas.py

```diff
@@ -463,6 +466,16 @@ def _parse_global_attrs(in_ds, out_ds):
     out_ds.attrs = in_ds.attrs
 
 
+def _parse_face_faces(in_ds, out_ds):
+    face_face_connectivity = in_ds["cellsOnCell"].values
+
+    out_ds["face_face_connectivity"] = xr.DataArray(
+        data=face_face_connectivity,
+        dims=ugrid.FACE_FACE_CONNECTIVITY_DIMS,
+        attrs=ugrid.FACE_FACE_CONNECTIVITY_ATTRS,
+    )
+
+
```

Comment:
Need to replace the padding & zeros (like the example below). Please add a docstring too. 

https://github.com/UXARRAY/uxarray/blob/3b27d3f3ccf1d4a84a28f6538b2a91f79e051160/uxarray/io/_mpas.py#L361-L395

Reply:
Is this connectivity included in the dual mesh? It seems like it wouldn't be to me, just wanted to double-check.

Reply:
Not natively (if there was, it'd need to be named "verticesOnVertex"), but it can be derived from the "edgesOnVertex" (i.e. the other vertex of each edge that is connected to the vertex of interest could give us "verticesOnVertex")